providers/edge3/src/airflow/providers/edge3/openapi/v2-edge-generated.yaml

```diff
@@ -919,6 +917,8 @@ components:
           - type: string
           - type: 'null'
           title: Log Path
+        ti:
```

Comment:
All the changes in this provider are because of a change in the order of the fields in ExecuteTask? The `ti` field was moved to the end because of a refactor to abstract common code to a super class